### PR TITLE
fix: Add `url` to `identifier` responses for MA ids

### LINKED ISSUE #1331: api endpoint `identifiers` contains incomplete response for MA ids
**Describe the bug**

In production environment.
The API response seems to slightly vary if the database name is `MetabolicAtlas` - it's missing the url which is otherwise returned for other databases.

**Steps to reproduce**

Running `http://localhost/api/v2/identifier/Protein%20Atlas/ENSG00000120915` returns a JSON containing
```
    "url": "https://identifiers.org/hpa:ENSG00000120915"
```
whereas running `http://localhost/api/v2/identifier/MetabolicAtlas/MAR02279` is missing the `url` to Identifiers.org.

**Expected behavior**

- [ ] the database name is consistent with the URL when visiting the `identifier` page
- [ ] `Metabolic Atlas` is provided as a suggestion in Swagger
- [ ] Tests verifying the fix are added

### PR DESCRIPTION
**Pull request by pdmnyberg**
**Related issue(s) and PR(s)**  
This PR closes #1331

It adds an identifier `url` to responses for `MetabolicAtlas` database items.

**Type of change**  
- [x] Bug fix (non-breaking change which fixes an issue)
 
**List of changes made**  
- Add `url` to responses with `dbName === 'MetabolicAtlas'`

**Example**
***command***
```sh
curl http://localhost/api/v2/identifier/MetabolicAtlas/MAR02279 | jq
```
***output***
```json
{
  "components": [
    {
      "id": "MAR02279",
      "model": "Fruitfly-GEM",
      "componentType": "Reaction",
      "version": "1.2.0"
    },
    {
      "id": "MAR02279",
      "model": "Human-GEM",
      "componentType": "Reaction",
      "version": "1.12.0"
    },
    {
      "id": "MAR02279",
      "model": "Mouse-GEM",
      "componentType": "Reaction",
      "version": "1.3.0"
    },
    {
      "id": "MAR02279",
      "model": "Rat-GEM",
      "componentType": "Reaction",
      "version": "1.3.0"
    },
    {
      "id": "MAR02279",
      "model": "Worm-GEM",
      "componentType": "Reaction",
      "version": "1.3.0"
    },
    {
      "id": "MAR02279",
      "model": "Zebrafish-GEM",
      "componentType": "Reaction",
      "version": "1.2.0"
    }
  ],
  "identifier": {
    "id": "MAR02279",
    "dbName": "MetabolicAtlas",
    "externalId": "MAR02279",
    "url": "https://identifiers.org/metatlas:MAR02279"
  }
}
```

**Testing**  
- Use command `ma-exec api npx jest test/identifier.test.js`
- Verify that the test looks sane

- Or `curl http://localhost/api/v2/identifier/MetabolicAtlas/MAR02279 | jq`
- Verify that `.identifier.url` exists and works

**Further comments**  
My expectation is that this should be enough to solve this task. As far as I could tell only the items relating to externalDbs included `url` in their data so this should not conflict with any table data. However I am open to the possibility that it could make sense to first check for an existing `url` on the object before applying the generated on.

**Definition of Done checklist**  
- [x] My code meets the Acceptance Criteria
- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code and commented any hard-to-understand areas
- [x] Tests and lint/format validations are passing
- [x] My changes generate no new warnings

### CODE REVIEW

File: api/src/swagger/config.yaml [hunk 1]
```diff
@@ -607,12 +607,12 @@ paths:
...
           required: true
           type: 'string'
-          description: 'the name of the database, e.g. **Protein Atlas**'
+          description: 'the name of the database, e.g. **Protein Atlas** or **MetabolicAtlas**'
         - name: 'externalId'
           in: 'path'
           required: true
           type: 'string'
-          description: 'the external ID, e.g. **ENSG00000120915**'
+          description: 'the external ID, e.g. **ENSG00000120915** or **MAR02279** for *Protien Atlas* and *MetabolicAtlas* respectively'
```
Comment (Jessica-Sandler) [inline api/src/swagger/config.yaml:615]: Just a small misspelling of Protein Atlas here

**Review by Jessica-Sandler — DISMISSED** [review]
Looks good! I have tested the changes in all the described ways and everything works as expected.

**Review by perjo — APPROVED** [review]
Looks good, well done! (typo pointed out by Jessica should be fixed)

**Review by jonandernovella — APPROVED** [review]
LGTM

**Pull request merged**